astro/src/content/docs/customize/look-and-feel/cli.mdx

```diff
@@ -3,6 +3,7 @@ title: FusionAuth CLI
 description: Learn about the FusionAuth admin CLI tool
 section: customize
 subcategory: look and feel
+# TODO: move this to top level customize as it can be used for email template management too
```

Comment:
Did you mean to do this, or just that it needs to be done in the future?  I think it is the second but just want to verify.

Reply:
Need to be done in the future. Didn't want too many moving pieces in this PR.